lib/ts/station/FSM.ex

```diff
@@ -114,6 +121,7 @@ defmodule Station.FSM do
   defstate process_query do
     defevent initialise,
       data: station_data = [itinerary, station_struct, dependency] do
+      Logger.info(fn -> "Current state='process_query'; Next state='ready'" end)
```

Comment:
This info type logger message can be moved from here to between lines-147 and 148. The message shall be "query <query_id> processing complete at station <station_name/id>".